modules/cnv_vm_console_web.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv_users_guide.adoc
+
+[[cnv-vm-console-web]]
+== Connecting to a virtual machine console
+
+You can connect to the VNC and serial consoles of a running virtual machine from the *Virtual Machine Details* screen of the web console.
+
+[NOTE]
+====
+When the *Disconnect before switching* checkbox is active, the console session is disconnected when you switch between consoles so that only one session is open at a time. Otherwise, the console sessions remain active in the background. 
+====
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machines*  from the side menu.
+. Select a Virtual Machine.
+. Click *Consoles*. The VNC console opens by default.
+. Use the *Console* drop-down list to change the console type. 
```

Comment:
Hm, there is no "Console" drop-down list. "VNC Console" and "Serial Console" are there.